libraries/aws_vpc_endpoint_connection_notification.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsVpcEndpointConnectionNotification < AwsResourceBase
+  name 'aws_vpc_endpoint_connection_notification'
+  desc 'Verifies settings for an AWS VPC Endpoint Notification'
+
+  example "
+    describe aws_vpc_endpoint_connection_notification(connection-notification-id: 'vpce-12345678123456789') do
+      it { should exist }
+    end
```

Comment:
Could you please add one more example like the below?
```suggestion
    describe aws_vpc_endpoint_connection_notification('vpce-12345678123456789') do
      it { should exist }
    end
```